### PR TITLE
Improvement/metrics table

### LINKED ISSUE #20: improve metrics table
add forecast_hour float (nullable) and ensemble_member int (nullable) to the expt_metrics table

change forecast_length to forecast_hour float in file counts table

add file_size_bytes bigint to file counts table

### PR DESCRIPTION
**Pull request by jrknezha**
Per discussions from feedback on table, this PR updates two tables: expt_metrics and expt_stored_file_counts as found in #20. 

Adds forecast_hour and ensemble_member to expt_metrics.
Changes the name from forecast_length to forecast_hour in expt_stored_file_counts and adds in the file_size_bytes column. 

Code updates:
* add the new columns to the tables 
* update the code which interacts with the tables to add these new fields
* update the harvest codes which have metric references 
* update the tests to reflect new changes and pass 

All tests are passing.

### CODE REVIEW

File: src/expt_file_counts.py [hunk 1]
```diff
@@ -75,7 +76,8 @@ class ExptFileCount:
     folder_path: String
     cycle: datetime
     time_valid: datetime
-    forecast_length: float 
+    forecast_hour: float 
+    file_size_bytes: int
```
Comment (frolovsa) [inline src/expt_file_counts.py:80]: is this int big enough? int32 is probably not but int64 is

Comment (jrknezha) [reply src/expt_file_counts.py:80]: yes, python only has one int size so it'll scale up appropriately. I double checked it.

File: tests/test_expt_file_counts_handler.py [hunk 2]
```diff
@@ -23,7 +23,8 @@ def test_put_expt_file_counts_request_dict():
             'file_type_name': 'example_type',
             'file_extension': '.example',
             'time_valid': '2023-02-05 06:00:00',
-            'forecast_length' : 120,
+            'forecast_hour' : 120,
+            'file_size_bytes': 1203949459,
```
Comment (frolovsa) [inline tests/test_expt_file_counts_handler.py]: can we make test input to be something that is bigger than a single precision float? Sorry i got lost in significant digits here. maybe it already is

**Review by frolovsa — CHANGES_REQUESTED** [review]
@jrknezha , thank you this looks good. a couple of small comments. Maybe they are nothing. just making sure we test with bigint.

File: tests/test_expt_file_counts_handler.py (hunk 2 above)
Comment (jrknezha) [reply tests/test_expt_file_counts_handler.py]: yes, I'll update it, it is short a digit.

**Review by frolovsa — APPROVED** [review]
thank you for addressing my comments! good to go

File: tests/test_expt_file_counts_handler.py (hunk 2 above)
Comment (jrknezha) [reply tests/test_expt_file_counts_handler.py]: updated it and confirmed the test is still working.

**Pull request merged**